proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -73,5 +74,10 @@ message Event {
   // Time of the audited event. It specifies "when" it happened.
   google.protobuf.Timestamp event_time = 5;
   // Payload of the audited event. It specifies additional context on the event.
-  // TODO add oneof payload events.
+  oneof payload {
+    OrganizationCreatedPayload organization_created = 6;
+  }
 }
+
+// OrganizationCreatedPayload is the payload for an ORGANIZATION_CREATED event.
+message OrganizationCreatedPayload {}
```

Comment:
I think it's fine to add this even if it's empty.

We could include the organization name?

Reply:
(also, feel free to suggest a better naming scheme for this or the `oneof`)

Reply:
Yeah, `name` and `create_time` seem useful (`update_time` I feel adds no value for this event).

Reply:
is `create_time` the same as `timestamp`?

Reply:
> is create_time the same as timestamp?

I'm assuming this represents when the event was actually written to whatever store it was accessed from.

if that's the case, then: 

> Yeah, name and create_time seem useful (update_time I feel adds no value for this event).

is it important to an audit log consumer know _when_ an event was written to `$STORE`? seems like an implementation detail (and I expect that would differ between `StdoutStore`, `GCSStore`, `PostgresStore`, etc...).

Reply:
I'd include `name` and `create_time`, even if [it's exactly the same value as `event_timestamp`](https://github.com/bufbuild/core/pull/5867#discussion_r961132030), just to be explicit.

Reply:
We could add both (and sorry, I misread, it's `event_time`, not `timestamp`). 

Maybe I'll clarify the docs that `event_time` is when the event is ... enqueued? recorded? (Not sure it matters, which is perhaps why I thought `event_time` could, in this case for example, be `Organization.created_at`, with no other timestamps.) wdyt @unmultimedio @njhale?

Reply:
(sorry, added that comment before I saw @unmultimedio's). I think that's reasonable. I'll add those fields.

Reply:
And now that `name` is back, we could have that name being the FQN, and still have separate fields for those in the payload?

```
{
  "event_time": "same as created_at",
  "actor" {
    "type": "USER",
    "name": "johndoe",
  },
  "resource" {
    "type": "ORGANIZATION",
    "name": "buf.build/acme", // for repositories/plugins/commits this can still be its FQN
  },
  "payload_organization_created": {
    "remote": "buf.build",
    "name": "acme",
    "owner": "johndoe", // do we have this?
    "created_at": "timestamp",
  }
}
```

Reply:
added in [b14b34e](https://github.com/bufbuild/buf/pull/1375/commits/b14b34e772acfca2cd58bcfbf3652279ce2d8ddb)

Reply:
yeah, I think FQN is best when available. (so probably no need for `"remote"` in this case? I think `"owner"` is implied?)

Reply:
for context, looked at FQN with @unmultimedio and decided against it for now in favor of just attaching names where we think it's reasonable (for example, on a "repository create", perhaps we'll add `organization_id` / `organization_name`.)